package/src/index.ts

```diff
@@ -1,634 +1,4 @@
-
-type AssetQuoteRequest =
-  | ({
-      assetKind: "nothing";
-    } & NothingAssetQuoteRequest)
-  | ({ assetKind: "transport" } & TransportAssetQuoteRequest)
-  | ({ assetKind: "fiat" } & FiatAssetQuoteRequest)
-  | ({ assetKind: "stock" | "crypto" } & MarketAssetQuoteRequest);
-
-type TradeAsset =
-  | ({
-      assetKind: "nothing";
-    } & NothingAsset)
-  | ({
-      assetKind: "other";
-    } & OtherAsset)
-  | ({ assetKind: "transport" } & TransportAsset)
-  | ({ assetKind: "fiat" } & FiatAsset)
-  | ({ assetKind: "stock" | "crypto" } & MarketTradeAsset);
-
-type Counterparty =
-  | ({ kind: "merchant" } & MerchantCounterparty)
-  | ({
-      kind: "user";
-    } & UserCounterparty)
-  | ({
-      kind: "service";
-    } & ServiceCounterparty);
-
-type TradeQuery = {
-  gainedAssetKind?: TradeQueryAssetKind;
-  lostAssetKind?: TradeQueryAssetKind;
-  counterpartyKind?: "merchant" | "user" | "service";
-} & InternalTradeQuery;
-
-type TradeQuote<
-  GainedAssetKind extends AssetKind,
-  LostAssetKind extends AssetKind
-> = {
-  gained: TradeAsset & { assetKind: GainedAssetKind };
-  lost: TradeAsset & { assetKind: LostAssetKind };
-  context: string;
-  expirationDateTime: string;
-};
-
-type TradeQueryAssetKind =
-  | "fiat"
-  | "stock"
-  | "crypto"
-  | "transport"
-  | "other"
-  | "nothing";
-
-type AssetAccount = {
-  serviceAccountID: string;
-  nickname: string;
-  linkedAccountID: string;
-  service: Service;
-} & (
-  | {
-      assetKind: "fiat";
-      accountKind: FiatMarketAccountKind;
-      currencyCode: string;
-      balance?: number;
-      ach?: ACHDetails;
-      wire?: WireDetails;
-    }
-  | {
-      assetKind: "stock" | "crypto";
-      accountKind: FiatMarketAccountKind;
-    }
-  | {
-      assetKind: "transport";
-      accountKind: TransportAccountKind;
-    }
-);
-
-type Trade = {
-  dateTime: string;
-  state: TradeState;
-  counterparty: Counterparty;
-  lost: TradeAsset;
-  gained: TradeAsset;
-};
-
-type TradeAssetRef =
-  | ({ assetKind: "transport" } & TransportAssetRef)
-  | ({ assetKind: "nothing" } & NothingAssetRef)
-  | ({ assetKind: "other" } & OtherAssetRef)
-  | ({ assetKind: "fiat" } & FiatAssetRef)
-  | ({ assetKind: "stock" | "crypto" } & MarketTradeAssetRef);
-
-type LinkedAccountDetail =
-  | (LinkedAccount & {
-      details: { state: "active" } & ActiveLinkedAccountDetails;
-    })
-  | (LinkedAccount & {
-      details: { state: "inactive" | "unavailable" };
-    });
-
-export type {
-  Address,
-  AppUser,
-  AssetAccount,
-  AssetAccountRef,
-  AssetKind,
-  Coordinates,
-  Counterparty,
-  LinkedAccount,
-  LinkedAccountDetail,
-  LinkedAccountRef,
-  LinkedAccountStatusRef,
-  Service,
-  Trade,
-  TradeAsset,
-  TradeAssetRef,
-  TradeQuery,
-  TradeQuote,
-  TradeState,
-};
+export { CandleContext, useCandle } from "./context";
+export type { CandleContextValue } from "./context";
+export { CandleProvider } from "./provider";
```

Comment:
Was all of the stuff in this file not needed before? Or did you change something that let it be cleaned up?